config/taxonomy/catalog.structs.schema.json

```diff
@@ -51,20 +55,42 @@
                                 "type": "string",
                                 "description": "name of column or component"
                             },
-                            "tags": { "$ref": "catalog.values.schema.json#/definitions/valid_metadata"}
```

Comment:
By removing the validation of the metadata aren't we are removing one of the key things the taxonomy is meant to address?  I.e. Ensuring that the metadata used in the catalog is the same metadata used in the policies?

Reply:
With this update, what we tried to achieve was to make tags as a map so that anything with a key,value pair can appear as a tag. This was actually suggested by @roee88 . Hence, that change was brought in. @simanadler

Reply:
Good catch. I suggest to move tags definition to catalog.structs.schema.json.

Reply:
@roee88 Do you suggest to keep the tags as map in catalog.structs.schema.json as it is now ?

Reply:
Yes, I think it will be fine for this first iteration.